inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/ClusterType.java

```diff
@@ -32,6 +32,7 @@ public class ClusterType {
     public static final String PULSAR = "PULSAR";
     public static final String DATAPROXY = "DATAPROXY";
     public static final String KAFKA = "KAFKA";
+    public static final String CUSTOM = "CUSTOM";
```

Comment:
please add a description for the `CUSTOM` type.

Reply:
It's a internal type,  should not be here,  I will delete it.